### PR TITLE
Home-Page added in mini project

### PR DESCRIPTION
**Pull request by ashifkhn**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,34 @@
 import "./App.css";
-import logo from "./logo.png";
+import NotesList from "./components/NotesList";
+import { useState } from "react"
+// import { nonoid } from "nanoid"
```
Comment (ravikumar0403) [inline src/App.js:4]: remove commented code if not required

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
 
 function App() {
```
Comment (ravikumar0403) [inline src/App.js:6]: This can also be changed to es6 syntax

File: src/components/NotesList.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import Notes from "./Notes";
+import AddNewNote from "./AddNewNote";
+import "./style/noteslist.css"
+
+
+const NotesList = ({ notes }) => {
+  console.log(notes)
```
Comment (ravikumar0403) [inline src/components/NotesList.js:7]: remove console logs

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and folder structure is also good, I've left a few sanity comments

File: src/App.css [hunk 4]
```diff
@@ -1,51 +1,7 @@
...
-  display: flex;
-}
-
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
-}
+.main_container{
+  max-width: 900px;
```
Comment (amirnathani9) [inline src/App.css:2]: try to add relative units

File: src/components/NotesList.js (hunk 3 above)
Comment (amirnathani9) [inline src/components/NotesList.js:7]: remove console

**Review by amirnathani9 — APPROVED** [review]
All the things are good.
- suggested some changes 
- suggestions are try to use relative units and try to do commits at every step

**Comment by Pratham82** [comment]
- Make the text area color same as the card background/transparent.
- Remove commented code
Suggestions:
- Keep the css file name same case as the component name.

**Review by Pratham82 — APPROVED** [review]

**Pull request merged**